py3status/modules/sysdata.py

```diff
@@ -233,17 +249,18 @@ def _calc_cpu_percent(self, cpu):
         cpu_used_percent = 0
         if cpu["total"] != self.last_cpu.get("total"):
             cpu_used_percent = (
-                1
-                - (
-                    (cpu["idle"] - self.last_cpu.get("idle", 0))
-                    / (cpu["total"] - self.last_cpu.get("total", 0))
-                )
-            ) * 100
+                   1
+                   - (
+                       (cpu["idle"] - self.last_cpu.get("idle", 0))
+                       / (cpu["total"] - self.last_cpu.get("total", 0))
+                   )
+               ) * 100
```

Comment:
No configuration file needed, plain defaults. I assume it reformatted for example `names` and `placeholders` lists because you added some values and it might think the line is too long now, but that's fine ;) As long as we all use the same formatter, we are fine.